test/fuzz/test_runner.py

```diff
@@ -251,16 +259,19 @@ def job(t, args):
         future.result()
 
 
-def run_once(*, fuzz_pool, corpus, test_list, src_dir, build_dir, use_valgrind):
+def run_once(*, fuzz_pool, corpus, test_list, src_dir, build_dir, use_valgrind, empty_min_time):
     jobs = []
     for t in test_list:
-        corpus_path = os.path.join(corpus, t)
+        corpus_path = corpus / t
         os.makedirs(corpus_path, exist_ok=True)
         args = [
             os.path.join(build_dir, 'src', 'test', 'fuzz', 'fuzz'),
             '-runs=1',
-            corpus_path,
         ]
+        if not any(corpus_path.iterdir()) and empty_min_time:
+            args.append(f"-max_total_time={empty_min_time}")
```

Comment:
this would still include `runs=1` which for an empty corpus would only generate 1 input.

```
Run process_messages with args ['/home/dergoegge/workspace/bitcoin/src/test/fuzz/fuzz', '-runs=1', '-max_total_time=10']INFO: Running with entropic power schedule (0xFF, 100).
INFO: Seed: 2455678762
INFO: Loaded 1 modules   (350793 inline 8-bit counters): 350793 [0x55c579a854b0, 0x55c579adaef9),
INFO: Loaded 1 PC tables (350793 PCs): 350793 [0x55c579adaf00,0x55c57a035390),
INFO: -max_len is not provided; libFuzzer will not generate inputs larger than 4096 bytes
INFO: A corpus is not provided, starting from an empty corpus
#2      INITED cov: 2324 ft: 2325 corp: 1/1b exec/s: 0 rss: 115Mb
#2      DONE   cov: 2324 ft: 2325 corp: 1/1b lim: 4 exec/s: 0 rss: 115Mb
Done 2 runs in 0 second(s)
```